.travis.yml

```diff
@@ -95,3 +104,9 @@ notifications:
 env:
   global:
      secure: "fdpCjdC0Qp/ZJqtrCHE4I/tHXWF1sORftm6khd6geqK7d4qWzIh6HzNN2BlF+2m4nyuyxo2wzPm/oGoqogVpBgrzpQ7SZl7h2/wzgs2C/k39sFGyDisLesTM5DhBDJWcomyqtcnQmKn340Z9KOxiHAt4FOj2FZVN5+tIO5j3Cks="
+
+## Cache dependencies
```

Comment:
Caches downloaded dependencies, so they are loaded from cache instead of full download